src/sentry/integrations/opsgenie/client.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from sentry.shared_integrations.client.proxy import IntegrationProxyClient
+
+OPSGENIE_API_VERSION = "/v2"
+
+
+class OpsgenieProxySetupClient(IntegrationProxyClient):
```

Comment:
For the setup client, there's actually no need to inherit from the `IntegrationProxyClient`. It's part of a hybrid cloud refactor but only needs to be used for the regular client (i.e. the one that will use the API to trigger alerts and notifs). The setup client can inherit directly from `ApiClient`

Reply:
You can also skip using `Proxy` in the name. Hybrid Cloud has abstracted the entire idea of requests being proxied away from the application code :) it should just happen behind the scenes.